wikimedia/wikipedia-ios
# Talk pages - change page language

**Phabricator:** (Partially) https://phabricator.wikimedia.org/T310294

### Notes
* This PR adds functionality to the talk page overflow menu option to change the language. It relies on existing components to allow the user to change to another language.

### Test Steps
1. Go to an article talk page
2. Select the change language option
3. A view controller with the available languages will show; select a language
4. Make sure all the content (topics, header, coffee roll) is now in the selected language
5. On a user talk page, select the change language option
6. On the new screen, select a language (you can add more languages at this moment if there's only one available)
7. Verify that the content is in the newly selected language
